package.json

```diff
@@ -7,7 +7,7 @@
   "typings": "./lib/index.d.ts",
   "author": "Azure Functions",
   "scripts": {
-    "start": "npm run test -- --watch",
+    "start": "jest --watch",
```

Comment:
why the change to trigger jest directly vs. the NPM command.  It doesn't seem like a big change but now `pretest` etc are also not run which did some linting, etc.

Reply:
Was going to ask your thoughts on this. I thought it was kind of nice to have a command to just trigger tests without having to wait for linting to finish. It was taking kind of a while for me, but I'm open to leaving this as is